oat-sa/tao-core
# fix: tr-296 fix scrollbars behavior for zoom (MSIE) in fullscreen mode

Related to:
- https://oat-sa.atlassian.net/browse/TR-296

vertical scrollbar added to page on zoom in fullscreen mode for MSIE

How to check:
- set `DEBUG_MODE` to  `false` in `config/generis.conf.php`
- check delivery in fullscreen mode with MSIE11, try to use zoom and ensure test runner controls are available
